packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -595,15 +598,10 @@ module('Integration | operator-mode', function (hooks) {
     );
     await waitFor('[data-test-person]');
     await click('[data-test-edit-button]');
+    onSave((json) => {
```

Comment:
In order to be sure that assertions inside a callback like this have been run, you need to call `assert.expect(N);` where N is the total number of assertions you expect to have run at the end of the test.